llama_index/llms/gemini.py

```diff
@@ -159,5 +160,25 @@ def stream_chat(
         merged_messages = merge_neighboring_same_role_messages(messages)
         *history, next_msg = map(chat_message_to_gemini, merged_messages)
         chat = self._model.start_chat(history=history)
-        it = chat.send_message(next_msg, stream=True)
-        yield from map(chat_from_gemini_response, it)
+        response = chat.send_message(next_msg, stream=True)
+
+        def gen() -> ChatResponseGen:
+            content = ""
+            for r in response:
+                top_candidate = r.candidates[0]
+                content_delta = top_candidate.content.parts[0].text
```

Comment:
is this guaranteed to always be the format? Just want to make sure we avoid indexing or Nonetype errors when accessing the data (if needed)👍🏻

Reply:
@markmcd can help confirm.
from my local testing, this is always the case to get delta from parts